problems/arraysIntro.js

```diff
@@ -64,29 +84,54 @@ function arrayToString() {}
  * @param {Array} array2
  * @returns {string|Array}
  */
-function largerArray() {}
+function largerArray(array1, array2) {
+    if (array1.length>array2.length){
+      return array1;
+    } else if (array2.length>array1.length){
+      return array2
+    } else if (array1.lenght===array2.lenght){
+      return 'They are the same size'
+    }
+  }
 
 /**
  * Takes in an array and returns the middle element.
  * If the array is even in length return the middle two elements in their own array.
  * @param {Array} array
  * @returns {*|Array}
  */
-function middleElement() {}
+function middleElement(array) {
+  if ( array.length % 2 !== 0){
+    let middleElement = Math.floor( array.length / 2);
+      return array[ middleElement ];
+  } else{
+      let lowMiddleElement = ((array.length / 2 ) -1)      
+      let highMiddleElement = (array.length / 2 )         
+      let middleValues = [array[lowMiddleElement], array[highMiddleElement]]
```

Comment:
I love that you saved these are individual elements. Makes for a very clean return. Line 109 can also be Math.floor so you can reuse line 105 can be outside the if statement  instead of writing line 109